manual_jb/content/_toc.yml

```diff
@@ -81,6 +81,7 @@ chapters:
     - file: notebooks/Joplin_testbed/Joplin_testbed.ipynb
     - file: notebooks/Lumberton_testbed/Lumberton_testbed.ipynb
     - file: notebooks/Seaside_testbed/Seaside_testbed.ipynb
+    - file: notebooks/MMSA_testbed/MMSA_Seismic_Functionality_Analysis_for_Interdependent_Buildings-Water-Power_using_Fragility_and_Repair_Rate_Curves.ipynb
```

Comment:
Can these MMSA notebooks be called MMSA_testbed-1.ipynb and MMSA_testbed-2.ipynb and the internal titles be the long names so it's consistent with the other notebooks?

Reply:
Jong said the long name is fine, but maybe later we consider if we should rename them.